docs/settings.rst

```diff
@@ -2038,6 +2038,16 @@ The value between the hooks is a UUID that is generated for each request. This c
 
 Maximum request/response body size in megabytes.
 
+.. _setting-webserver-connection-timeout:
+
+``webserver-connection-timeout``
+--------------------------
+
```

Comment:
```suggestion
.. versionadded:: 5.0.0

```

Reply:
I committed the suggestion, but on a second thought would it be possible to have this in 4.8.x? I can back port the changes to 4.8.x and 4.9.x

Reply:
yes, but the default behaviour has to remain unchanged

Reply:
like https://doc.powerdns.com/authoritative/settings.html#lua-records-insert-whitespace